### DISCUSSION TITLE
OTP fields ?

### DISCUSSION THREAD

**Discussion by pravin-d** [danvick/flutter_form_builder#1310] [Q&A]
I want to implement a screen that is for otp. The idea is that it will contain a single input box for each number/character. Starting from the first box, when the user enters a number, it will just jump to the next input box. Seems like a standard usage in most app. So I was wondering if form builder already has a similar implementation ?

**Comment by deandreamatias** [comment]
No exist a implementation about this. You can use 6 FormBuilderTextField with numeric configuration or use a external package, from [this list](https://pub.dev/packages?q=otp) for example

**Reply by pravin-d** [reply]
I checked the other solutions. They seem to use core flutter form elements. If there is an implementation using FormBuilder, would you include it as part of the eco-system ?
